ajax/libs/enzyme/2.2.0/Debug.js

```diff
@@ -0,0 +1,157 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+
+var _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function (obj) { return typeof obj; } : function (obj) { return obj && typeof Symbol === "function" && obj.constructor === Symbol ? "symbol" : typeof obj; };
+
+exports.typeName = typeName;
+exports.spaces = spaces;
+exports.indent = indent;
+exports.debugNode = debugNode;
+exports.debugNodes = debugNodes;
+exports.debugInst = debugInst;
+exports.debugInsts = debugInsts;
+
+var _ShallowTraversal = require('./ShallowTraversal');
```

Comment:
That will be ok and users can use a shim if they need to - http://stackoverflow.com/questions/10282075/object-defineproperty-for-all-browsers